cat_merge/merge.py

```diff
@@ -45,7 +45,7 @@ def merge(
   mappings: {mappings}
   output_dir: {output_dir}
 """)
-    if nodes is None != (edges is None) or source is None == (nodes is None):
```

Comment:
@amc-corey-cox This was failing for me when passing in just source, I _think_ my simplification should still be ok?